### ISSUE TITLE
A short history of logarithms; or how to multiply large numbers

### ISSUE THREAD

**Issue by mushunrek** [leios/SoME_Topics#151]
## About the author

I am about to start my PhD in Mathematics in October and have some spare time left during summer. I thought about this project last summer when I first heard about this amazing environment, but I don't have the necessary skills (or the time to get good enough) to finish the project on my own. And anyways, it is always more interesting to get to know new people in the process. :)

## Quick Summary

The aim of the project is to show logarithms from a different point of view from the one usually taken in school classes. Instead of introducing it a) to solve equations involving an exponential or (less frequently) b) as the area beneath the hyperbola, we would take a historical approach. It turns out that logarithms were first invented to simplify difficult multiplications (for sailors and astronomers) by transforming multiplications into additions. At the end of the video, viewers should be able to use logarithmic tables. If some time is left, it would be nice to draw a connection to the points a) and b) viewers see in or might remember from school.

TL;DR: the project shows (heuristically and on an elementary level) how to get to logarithms through the functional equation $$f(xy) = f(x) + f(y).$$

## Target audience

I have two types of audiences in mind. Since I  am not convinced that most highschool students would watch this type of video, it is not meant as an aid for school. Instead, it is meant a) for highschool students that are dissatisfied with the introduction of the logarithm in school and would like to get to it from another point of view and b) for university students (or the wider public) who might be unfamiliar with this amazing story behind logarithms.
The only important thing: the video should be easy enough to understand without any advanced knowledge in mathematics. 

## Target medium

I thought about doing a video which could be uploaded to YouTube. When I worked on this project, I concentrated on using Manim as I can program in Python and this seemed the easiest entry into video creation/editing. But if you can do other and probably more suited stuff to animate and illustrate the above: I am open to everything!!
The language of the video would probably English. If you are motivated to add caption, I would be able and glad to translate the final script into German and French.

### More details

There a very nice [article](https://lewebpedagogique.com/h4mathsts1/files/2013/12/105_122_AM61-4-1.pdf) (unfortunately only French), where the historic development is discussed.

I have already developed a storyline, but I guess that it still has to be finetuned. If you are interested, it is probably easiest to discuss all of that face to face (or most likely via Zoom ^^).

### Contact details

If you are interested in collaborating, you can contact me in the comments below or via mail: julian.kern@ens-lyon.fr

**Comment by Dayn-Hues** [comment]
I don't think I have the content producing skills to do what you are saying but I just wanted to say I really like your idea and I hope it gets made into a full video!!
